projects/netcdf/patch.diff

```diff
@@ -0,0 +1,46 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Can you please get these patches landed upstream, diff like these will break soon.

Reply:
https://github.com/Unidata/netcdf-c/pull/1972